### ISSUE TITLE
Unable to build Tiled on Windows in Debug mode

### ISSUE THREAD

**Issue by bjorn** [mapeditor/tiled#3647]
When building Tiled in Debug mode on Windows, the following error pops up:

```
:-1: error: Cannot install files
'C:/Projects/build-tiled-Desktop_Qt_6_4_2_MSVC2019_64bit-Debug/Debug_Desktop__49d835fd610c6acf/tiled.1920fdf8/tiled.pdb' and
'C:/Projects/build-tiled-Desktop_Qt_6_4_2_MSVC2019_64bit-Debug/Debug_Desktop__49d835fd610c6acf/libtiled.080f31f8/tiled.pdb'
to the same location
'C:/Projects/build-tiled-Desktop_Qt_6_4_2_MSVC2019_64bit-Debug/Debug_Desktop__49d835fd610c6acf/install-root/tiled.pdb'.
If you are attempting to install a directory hierarchy, consider using the qbs.installSourceBase property.
```

This happens as a result of 46c447422209929b14e6622dec51942d5bbe09b1, since before that the debug information files were not being installed at all.

Apparently `tiled.exe` and `tiled.dll` have the same separate debug information file `tiled.pdb`, they can't install to the same location. We'd need to either resolve the naming conflict or disable installing debug information files (though they are likely necessary for debugging to work properly).

**Comment by azagaya** [comment]
Changing the `targetName` in libtiled.qbs to something like "libtiled" instead of "tiled" seems to fix this, and debug works with that change. Is there any problem with that to your knowledge?

**Comment by bjorn** [comment]
@azagaya I guess there isn't really a problem with it, apart from that rather uncommon resulting DLL name "libtiled.dll". And of course such a rename will need a small adjustment in the installer. I have no objections to this solution.

**Comment by bjorn** [comment]
Resolved by #3832.
